packages/contracts/contracts/core/IDAO.sol

```diff
@@ -2,9 +2,11 @@
 
 pragma solidity 0.8.10;
 
-/// @title The interface required to have a DAO contract within the Aragon DAO framework
+/// @title IDAO
 /// @author Aragon Association - 2022
+/// @notice The interface required for DAOs within the Aragon App DAO framework.
```

Comment:
I'd add something about only DAOs with this interface will be registered in Aragon's DAO Registry.

Reply:
Isn't this information more relevant for the `DAORegistry` contract and should be given there?